opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -201,15 +187,6 @@ impl Debug for BatchLogProcessor {
 
 impl LogProcessor for BatchLogProcessor {
     fn emit(&self, record: &mut LogRecord, instrumentation: &InstrumentationScope) {
```

Comment:
if an emit is called after shutdown, even if we don't do is_shutdown check, the channel would be already closed, so the error it returned is good enough?

Reply:
something like
1. Don't check for is_shutdown.
2. Just export as usual.
3. Since the channel is closed, it'll error out.
4. Log that error.

No contention/perf cost for normal path. If logs are still emitted after shutdown, it clearly indicates some issue with user managing the lifetimes.